src/lib/components/footer.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+	import { AppBar } from '@skeletonlabs/skeleton';
+</script>
+
+<AppBar>
+    <svelte:fragment slot="lead">
+        <a
+            class="btn btn-sm variant-ghost-surface"
+            href="https://github.com/Samoa-Stack-Overflow"
+            target="_blank"
+        >
+            GitHub
+        </a>
+        &nbsp;&nbsp;
+        <a
+            class="btn btn-sm variant-ghost-surface"
+            href="https://samoa-stack-overflow.atlassian.net/browse/WEB"
+            target="_blank"
+        >
+            Jira
+        </a>
```

Comment:
Can the link and title be changed. I wouldn't want the public to see our JIRA board and tickets lol

Reply:
Instead you can use Discord as title and link can be # for now, while I sort out a survey that people have to answer before getting sent a link to the discord.

Reply:
or use GitHub i.e. https://github.com/Samoa-Stack-Overflow

Reply:
Done. See latest commit.